crainets/essentials/multi_tracker.py

```diff
@@ -112,6 +116,48 @@ def training_metric(self, epoch):
     def validation_metric(self, epoch):
         return self.results[self.VALIDATION_KEY][epoch]
 
+    def plot(self, show: bool = True, save_path: Optional[Union[str, Path]] = None):
+        """Plot validation and training losses
+
+        Args:
+            show (bool, optional): Whether to show the plot or not. Defaults to True.
+            save_path (Optional[Union[str, Path]], optional): The path where the training and validation plots are shown. Defaults to None.
+        """
+        train = self.training()
+        valid = self.validation()
+        training = list()
+        validation = list()
+        x = np.arange(len(train.items()))
+
+        for k, (i, j) in enumerate(train.items()):
+            training.append(np.mean(np.array(j['loss'])))
+
+        for k, (i, j) in enumerate(valid.items()):
+            validation.append(np.mean(np.array(j['loss'])))
+
+        cm = 1/2.54
```

Comment:
I would give these vars a more explicit name and place them in config instead. It clutters the code